Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+pipeline {
+  agent {
+    label 'x86&&macOS&&Apps'
+  }
+  environment {
+    VIEW = 'dsp'
+    REPO = 'lib_dsp'
+  }
+  options {
+    skipDefaultCheckout()
+  }
+  stages {
+    stage('Get view') {
+      steps {
+        prepareAppsSandbox("${VIEW}", "${REPO}")
+      }
+    }
+    stage('Library checks') {
+      steps {
+        libraryChecks("${REPO}")
+      }
+    }
+    stage('App Notes') {
+      steps {
+        buildAllApps("${REPO}/examples")
```

Comment:
This will currently fail as there are no non app note based example apps to build - see https://github.com/xmos/xmos_jenkins_shared_library/issues/54.

Reply:
As https://github.com/xmos/xmos_jenkins_shared_library/pull/56 has now been merged you will need to call `xcoreAllAppsBuild()` here.